addons/sourcemod/scripting/include/surftimer.inc

```diff
@@ -168,7 +185,7 @@ forward Action surftimer_OnCheckpoint(int client, float fRunTime, char sRunTime[
  * @param client			The client's ID.
  * @return						Plugin_Handled, to stop the player from searching.
  */
-forward Action surftimer_OnBonusFinished(int client, float fRunTime, char sRunTime[54], int rank, int total, int bonusid);
+forward Action surftimer_OnBonusFinished(int client, float fRunTime, char sRunTime[54], float fPBDiff, float fSRDiff, int rank, int total, int bonusid, int style);
```

Comment:
What's the reason for this change? Plugins which use the old version, will not work anymore.

Reply:
same reason as `onmapfinished`